llama_index/postprocessor/sbert_rerank.py

```diff
@@ -10,8 +10,16 @@
 
 
 class SentenceTransformerRerank(BaseNodePostprocessor):
-    model: str = Field(ddescription="Sentence transformer model name.")
+    model: str = Field(description="Sentence transformer model name.")
     top_n: int = Field(description="Number of nodes to return sorted by score.")
+    device: str = Field(
+        default=infer_torch_device(),
```

Comment:
Instead of calling the function here, let's let the contructor call it (to avoid dependency issues with torch).

We can skip the default here, or set it to "cpu"

Reply:
make sense. Will update.